src/client/datascience/interactive-ipynb/trustCommandHandler.ts

```diff
@@ -77,6 +77,7 @@ export class TrustCommandHandler implements IExtensionSingleActivationService {
             default:
                 break;
         }
+        return selection === DataScience.trustNotebook();
```

Comment:
Are trustNativeNotebook and trustNotebook guaranteed to work or throw? Just making sure here as we're basing the return on the users selection, not the result of the trust operation.

Reply:
Good catch (no pun intended). The part where we stringify the notebook contents for trusting may throw if a circular reference is detected in the notebook JSON (which shouldn't happen). The rest of those methods as currently implemented won't throw but there's nothing enforcing that right now, so I'll add a catch for errors from calling those methods.